src/parser/postgresparser.cpp

```diff
@@ -454,12 +452,12 @@ std::unique_ptr<AbstractExpression> PostgresParser::AExprTransform(A_Expr *root)
 }
 
 // Postgres.BoolExpr -> terrier.ConjunctionExpression
-std::unique_ptr<AbstractExpression> PostgresParser::BoolExprTransform(BoolExpr *root) {
+std::unique_ptr<AbstractExpression> PostgresParser::BoolExprTransform(ParseResult *parse_result, BoolExpr *root) {
   std::unique_ptr<AbstractExpression> result;
-  std::vector<std::shared_ptr<AbstractExpression>> children;
+  std::vector<std::unique_ptr<AbstractExpression>> children;
   for (auto cell = root->args_->head; cell != nullptr; cell = cell->next) {
     auto node = reinterpret_cast<Node *>(cell->data.ptr_value);
-    children.emplace_back(ExprTransform(node));
+    children.emplace_back(ExprTransform(parse_result, node, nullptr));
```

Comment:
should this be move(ExprTransform) ? Doesn't emplace_back make a constructor call and unique_ptrs aren't copiable so this has to be an explicit move constructor.

Reply:
Doesn't emplace_back use std::forward? https://en.cppreference.com/w/cpp/container/vector/emplace_back